### PR TITLE
Update Eline's Section in AboutUs.md

### LINKED ISSUE #26: Update Eline's Section in AboutUs.md

### PR DESCRIPTION
**Pull request by Elinengu**
Closes #26

### CODE REVIEW

**Comment by lithops-zty** [comment]
lgtm!

**Review by weiseng18 — APPROVED** [review]
👍

File: docs/AboutUs.md [hunk 1]
```diff
@@ -13,49 +13,9 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ### John Doe
```
Comment (lithops-zty) [inline docs/AboutUs.md:14]: change to your name?

**Review by MAOXIONGKAI — APPROVED** [review]
LGTM!

**Review by weiseng18 — APPROVED** [review]
👍

**Review by MAOXIONGKAI — APPROVED** [review]
LGTM!

**Pull request merged**